app/controllers/events/view/tickets/access-codes/list.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'code',
+      title        : 'Access Code'
+    },
+    {
+      propertyName   : 'accessUrl',
```

Comment:
@CosmicCoder96 If we disable the filter/sort then the field is not included in the query. Ideally it should be `access-url`, but I guess it was not working here(nothing is displayed)